metron-interface/metron-alerts/src/app/alerts/alerts-list/auto-polling/auto-polling.service.spec.ts

```diff
@@ -442,7 +442,7 @@ describe('AutoPollingService', () => {
 
   });
 
-  describe('polling state persisting and restoring', () => {
+  fdescribe('polling state persisting and restoring', () => {
```

Comment:
It looks like a focus was accidentally left on this spec.

Reply:
Done, Thanks